packages/create-plugin/templates/common/.config/types/webpack-rules.d.ts

```diff
@@ -25,8 +25,8 @@ declare module '*.webp' {
 }
 
 declare module '*.svg' {
-  const content: string;
-  export default content;
+  const src: string;
```

Comment:
I'd rather not rename this file as I don't think these declarations are specific to Webpack. Any project that uses a bundler that handles importing of images / fonts would benefit from them.

Reply:
Would you be amenable to `bundler-rules.d.ts` or similar? I found `custom.d.ts` to be too generic when there are now also types for actual modules in here as well?

Reply:
I guess that's a better name for it, yeah.